### PR TITLE
[python] Make docs searchable

### PR DESCRIPTION
**Pull request by ivirshup**
This PR addresses https://github.com/chanzuckerberg/cellxgene-census/issues/620

I've re-enabled Sphinx's built-in client side search by:

* Removing the blank template for the search bar
* Removing the line:

```python
html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
```

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -66,9 +66,7 @@
 # Inject custom css files in `/_static/css/*`
 html_static_path = ['_static']
 
-import sphinx_rtd_theme
 html_theme = "sphinx_rtd_theme"
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
```
Comment (ivirshup) [inline docs/conf.py:71]: @ebezzi, do you remember why you added this?

Comment (ebezzi) [reply docs/conf.py:71]: I believe that was a requirement for RTD.

**Review by ebezzi — APPROVED** [review]

**Review by pablo-gar — APPROVED** [review]
Looks great. Thanks @ivirshup! We'll need to check it works well once deployed in `github.io`

**Comment by ivirshup** [comment]
Yeah, it would be nice if we had preview-able doc builds as part of CI. Also a hosted development version of docs.

**Pull request merged**